modules/cart/src/CartManager.php

```diff
@@ -56,6 +64,7 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Lin
     $this->lineItemStorage = $entity_type_manager->getStorage('commerce_line_item');
     $this->lineItemMatcher = $line_item_matcher;
     $this->eventDispatcher = $event_dispatcher;
+    $this->languageManager = \Drupal::languageManager();
```

Comment:
Update `services.yml` to inject this into the service. Update `create` method to get it from service, make it an argument.